packages/tailwind/index.ts

```diff
@@ -9,11 +9,11 @@ import plugin from 'tailwindcss/plugin';
  * - Links Starlight’s fonts to `sans` and `mono` in Tailwind theme settings.
  *
  * @example
- * // tailwind.config.cjs
+ * // tailwind.config.mjs
  * const colors = require('tailwindcss/colors');
  * const starlightPlugin = require('@astrojs/starlight/tailwind');
```

Comment:
```suggestion
 * import colors from 'tailwindcss/colors';
 * import starlightPlugin from '@astrojs/starlight-tailwind';
```

Note: this Starlight Tailwind plugin import in this comment also had a typo.